packages/core/template/portfolio/_markbind/layouts/default.md

```diff
@@ -0,0 +1,34 @@
+<head-bottom>
+  <link rel="stylesheet" href="{{baseUrl}}/stylesheets/main.css">
+</head-bottom>
+
+<header sticky>
+  <navbar type="dark">
+    <a slot="brand" href="{{baseUrl}}/index.html" title="Home" class="navbar-brand">Your Name</a>
+    <li><a href="{{baseUrl}}/index.html#skills" class="nav-link">Skills</a></li>
+    <li><a href="{{baseUrl}}/index.html#experience" class="nav-link">Experience</a></li>
+    <dropdown header="Projects" class="nav-link">
+      <li><a href="{{baseUrl}}/index.html#project-1" class="dropdown-item">Project 1</a></li>
+      <li><a href="{{baseUrl}}/index.html#project-2" class="dropdown-item">Project 2</a></li>
```

Comment:
Could add the other projects here also?

Reply:
As in, add "Other Projects" - not indiv links to all the other projects